src/app/modules/core/core.component.html

```diff
@@ -25,7 +25,7 @@
         </div>
         <hr>
         <div class="hovestyle">
-            <a class="mainsidenavlist">Logout</a>
+            <a class="mainsidenavlist" (click)="logout()">Logout</a>
```

Comment:
this class is used in side nav style for so many links. logout is just one link in side nav. so we can not rename class to logout